usl_pipeline/usl_lib/usl_lib/readers/polygon_readers.py

```diff
@@ -0,0 +1,49 @@
+import typing
+from typing import Tuple
+
+from shapely import geometry
+
+
+def read_polygons_from_text_file(
+    file: typing.TextIO, support_mask_values: bool = False
+) -> list[Tuple[geometry.Polygon, int]]:
+    """Reads polygon information with optional mask values from text file.
+
+    Args:
+        file: File text stream to load from.
+        support_mask_values: Optional indicator of the presence of polygon mask values
```

Comment:
Should we handle this automatically? As in, when reading the file, check to see if mask values are present, and if so returning them? Is there a reason why callers would want to have support_mask_values as False, even if mask values are present?

Reply:
Sure. Let me do it...